index.js

```diff
@@ -363,11 +364,11 @@ function replaceRequireAndDefine(code, amdPackages, amdModules) {
         }
 
         // Dealing with ember-auto-import eval
-        if (node.callee.name === 'eval') {
+        if (node.callee.name === 'eval' && node.arguments[0].type === 'Literal' && typeof node.arguments[0].value === 'string') {
           const evalCode = node.arguments[0].value;
           const evalCodeAfter = replaceRequireAndDefine(evalCode, amdPackages, amdModules);
           if (evalCode !== evalCodeAfter) {
-            write(buffer, "eval(\"" + evalCodeAfter.replace(/\n/g, "\\n") + "\");", node.range[0]);
+            write(buffer, "eval(" + JSON.stringify(evalCodeAfter) + ");", node.range[0], node.range[1] - node.range[0]);
```

Comment:
Are you sure about this?
Don't we need the double quote around the string?
`eval("....");`

Reply:
100% sure they are not needed. Tested in my application.

Idea gotten for here
https://stackoverflow.com/questions/21672334/javascript-how-to-show-escape-characters-in-a-string

Reply:
Yep you are right.